dist/origin-web-common-services.js

```diff
@@ -495,7 +500,7 @@ angular.module('openshiftCommonServices')
       var preferredVersion = defaultVersion[group.name] || group.preferredVersion;
       _.each(group.versions[preferredVersion].resources, function(resource) {
         // Exclude subresources and any rejected kinds
-        if (resource.name.indexOf("/") >= 0 || _.contains(rejectedKinds, resource.kind)) {
+        if (_.contains(resource.name, '/') || _.find(rejectedKinds, {kind: resource.kind})) {
```

Comment:
Tinkering with this, atm it is greedy, dropping both `Binding` and `{ kind: "Binding", group: "servicecatalog.k8s.io" }`.